backend/app/store/image/bolt_store.go

```diff
@@ -118,6 +119,16 @@ func (b *Bolt) Load(id string) ([]byte, error) {
 	return data, nil
 }
 
+// Delete image from storage
+func (b *Bolt) Delete(id string) error {
+	return b.db.Update(func(tx *bolt.Tx) error {
+		// deleting non-existing key doesn't return an error
+		err := tx.Bucket([]byte(imagesBktName)).Delete([]byte(id))
```

Comment:
combo of this code and the comment above is confusing. If deleting non-existing key doesn't return an error, why we bother to assign it to err? If it is needed, why we ignore (overwrite) the first error?

Reply:
I'll adjust it to say:
```
		// deleting a non-existing key doesn't return an error, so joining errors from deleting an image
		// from both buckets is safe and will return nil if there are no errors on the real delete
		// or image is absent in both buckets
```
My bad for not making the intention clear first.